### ISSUE TITLE
A bug in doc

### ISSUE THREAD

**Issue by lphcreat** [great-expectations/great_expectations#8320]
**Describe the bug**
a bug in doc

**Expected behavior**
https://docs.greatexpectations.io/docs/guides/expectations/advanced/how_to_compare_two_tables_with_the_onboarding_data_assistant/
![image](https://github.com/great-expectations/great_expectations/assets/28752787/527c8c2f-022f-4b86-a4f9-fc01353694df)
the code should be
```
data_assistant_result = context.assistants.onboarding.run(
    batch_request=mysql_batch_request, exclude_column_names=["VendorID"]
)
```

**Additional context**

**Comment by austiezr** [comment]
Hey @lphcreat ! Thanks for surfacing; we'll resolve asap.

**Comment by kwcanuck** [comment]
[PR 8331](https://github.com/great-expectations/great_expectations/pull/8331)

**Comment by austiezr** [comment]
@lphcreat closing this out as resolved by #8331. 🙇
